src/pages/LabelFavorites/index.tsx

```diff
@@ -1,9 +1,14 @@
-import { Divider, Flex, Img, Link, Text } from "@chakra-ui/react";
-import PlusIcon from "../../icons/PlusIcon";
-import QuestionMarkIcon from "../../icons/QuestionMarkIcon";
+import { Divider, Flex } from "@chakra-ui/react";
+import { useState } from "react";
+import NewTask from "../../components/Navbar/Components/NewTask";
+import { Reminder } from "../TodayPage/Components/types";
+import FavoritePage from "./components/FavoritePage";
 import Header from "./components/Header";
 
 function LabelFavorite() {
+  const [loading, setLoading] = useState(true);
+  const [reminders, setReminders] = useState<Reminder[]>([]);
+  const [showWelcome, setShowWelcome] = useState(true);
```

Comment:
Those should not be here. You should have defined those variables in the NewTask component. Remove those props.